tests/integrations/test_urlscan.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import pytest
+from httpx import Response
+
+from tracecat.integrations.urlscan import analyze_url
+
+
+@pytest.fixture
+def urlscan_secret(create_mock_secret) -> dict[str, str | bytes]:
+    mock_secret = create_mock_secret(
+        "urlscan", {"URLSCAN_API_KEY": os.environ["URLSCAN_API_KEY"]}
+    )
+    serialized_secret = mock_secret.model_dump()
```

Comment:
so based on the above comment, you need to perform the complementing deserialization (using the getter) to get the actual decoded values